server/src/ankaios_server/server_state.rs

```diff
@@ -209,14 +176,10 @@ impl ServerState {
 
     // [impl->swdd~agent-from-agent-field~1]
```

Comment:
About this requirement I am not sure. It is existing. The function returns the workloads specified for a specific agent. The req text talks about "... select the agent ... based on the agent field...". I would like to introduce into this req that the it selects rendered workloads as a comment or in the req body itself, but do not know if this fits really to the existing req text.